src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -120,7 +118,7 @@ Please consult your promise library documentation on how to hook into its global
 &nbsp;
 ## Manually Trigger Errors
 
-**[Mimi note: Best for when an app is in a broken state, but there are no explicit erros being thrown. So, you want to trigger an error, possibly with extra context passed in for debugging purposes.]**
+**[Mimi note: Best for when an app is in a broken state, but there are no explicit errors being thrown. So, you want to trigger an error, possibly with extra context passed in for debugging purposes.]**
 
 **[Mimi note: Yes, there's duplicate info. Still thinking on this. Trying to cater to the kind of dev who has a set flow. Usually, 1. Initialize, 2. optionally configure, 3. Manually trigger an error (because you wanna take it out for a test drive)]**
```

Comment:
I think that's pretty common. Let's minimize the duplication but not stress it as long as the duplicated sections link to each other and it's clear that one is the advanced section.

Reply:
Cool. Linked it and reduced duplication.